### PR TITLE
USHIFT-1117: migrate reboot  tests to RF

### PR DESCRIPTION
**Pull request by eslutsky**
this PR migrate the reboot test to the RF framework.

this test should:
-  create some  pod with a volume PV.
-  reboot the node
-  making sure the pod is still functional .

### CODE REVIEW

File: test/resources/oc.resource [hunk 1]
```diff
@@ -52,3 +52,10 @@ Oc Wait For
     ${timeout_opt}=    Set Variable    --timeout\=${timeout}
     ${output}=    Run With Kubeconfig    oc wait -n ${NAMESPACE} ${type_name} --for=${wait_for} ${timeout_opt}
     RETURN    ${output}
+
+Oc Create
+    [Documentation]    Run 'oc create' on a specific resource in the curret test namespace
+    ...    Returns the command's combined STDOUT/STDER
+    [Arguments]    ${opts}
```
Comment (ggiguash) [inline test/resources/oc.resource:59]: The format of arguments in this function is not consistent to the other similar functions in this file. 
Can we add concrete options, rather then one generic `opts`?

File: test/suites/reboot.robot [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Resource            ../resources/common.resource
+Resource            ../resources/oc.resource
+Resource            ../resources/ostree-health.resource
+
+
+*** Variables ***
+${SOURCE_POD}        ./assets/reboot/pod-with-pvc.yaml
+${POD_NAME_STATIC}    test-pod
+${NAMESPACE}          default
```
Comment (ggiguash) [inline test/suites/reboot.robot:12]: Would it be better to create a dedicated namespace for this test? 
We've got `Create Random Namespace` function in `kubeconfig.resource.

Comment (eslutsky) [reply test/suites/reboot.robot:12]: i can use `Suite Setup         Setup Suite With Namespace` at the beginning ..

File: test/suites/reboot.robot [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+${SOURCE_POD}        ./assets/reboot/pod-with-pvc.yaml
+${POD_NAME_STATIC}    test-pod
+
+
+*** Test Cases ***
+Reboot Test
+    [Documentation]    Run a reboot test and make sure pod with PVC remains functional.
+    [Setup]    Test Case Setup
+    Reboot MicroShift Host
+    Wait For Healthy System
```
[the remark this answers is not in the document]
Comment (eslutsky) [reply test/suites/reboot.robot:23]: Okay.. then i guess in the e2e tests we are using a non-ostree system.

**Comment by ggiguash** [comment]
This looks good. 
I'm putting it on hold until we refactor the "standard" suite tests because the current scenario is too long and this test adds 5.5 more minutes.
/lgmt 
/hold

**Comment by eslutsky** [comment]
@ggiguash , can we merge this ?

File: test/suites/standard/reboot.robot [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+${SOURCE_POD}           ./assets/reboot/pod-with-pvc.yaml
+${POD_NAME_STATIC}      test-pod
+
+
+*** Test Cases ***
+Reboot Test
+    [Documentation]    Run a reboot test and make sure pod with PVC remains functional.
+    [Setup]    Test Case Setup
+    Reboot MicroShift Host
+    Wait Until Greenboot Health Check Exited
```
Comment (ggiguash) [inline test/suites/standard/reboot.robot:23]: Let's change this to `Wait For Healthy System`?

Comment (eslutsky) [reply test/suites/standard/reboot.robot:23]: this keyword is shared with  different suits, i wouldn't want to refactor them all in this PR, we can open refactor PR later.

**Comment by eslutsky** [comment]
@ggiguash , do we also need to override `ci/prow/e2e-openshift-conformance-reduced` ?

**Pull request merged**